examples/personas-bank/versions.tf

```
@@ -0,0 +1,21 @@
+terraform {
+  required_version = ">= 0.13"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.10"
+    }
+    random = {
+      source  = "hashicorp/random"
+      version = "~> 3.1"
+    }
+    template = {
+      source  = "hashicorp/template"
+      version = ">= 2.1.2"
```

We should keep all versions pinned at the minor level to provide flexibility for the calling configurations.

```suggestion
      version = "~> 2.1"
```

---

Thanks, @aaron-lane, good eye. I meant to make a note about that asking @rogeruiz if there was any specific reason to pin to the patch version. Roger, do you know if it's safe to pin to `~> 2.1`? Any bugs in 2.1.0-1?

---

It should be safe, I'm not sure why it would have been pinned to an exact minor version @anniehedgpeth but 🤷🏽

---

Thanks, Roger!